integration-box/cuenote-import/get_logs.py

```diff
@@ -0,0 +1,45 @@
+import cuenote
+import pandas
+import io
+import os
+import sys
+
+os.system(f"{sys.executable} -m pip install -U pytd==1.0.0 td-client")
+import pytd
+
+TD_API_KEY = os.environ.get('td_apikey')
+TD_API_SERVER = os.environ.get('td_endpoint')
+TD_DATABASE = os.environ.get('td_database')
+
+
+def main():
+    # Create a TD client instance.
+    client = pytd.Client(apikey=TD_API_KEY, endpoint=TD_API_SERVER, database=TD_DATABASE)
+
+    # Download log files from Cuenote, then upload CSVs to TD
+    expids = client.query("SELECT expid FROM queue")
```

Comment:
If the data could be huge, you might be able to use `cursor.fetchone()` with python-presto-client https://pytd-doc.readthedocs.io/en/latest/usage.html#working-with-db-api, which loads data in a streaming manner.

Reply:
This "queue" is a waiting list for report generation by Cuenote, and expid is created per a delivery setting for the last 14 days, so # of expid won't be huge. (is not # of recipient)